Docs/user_guide/modules/terminologies.md

```diff
@@ -0,0 +1,47 @@
+# Terminologies
+
+## Overview
+
+The Terminologies module provides a coding system for specifying anatomy, clinical findings, or other clinical terms. For example, the coding system can be used to specify what anatomical parts are stored in a segmentation node. Using standard codes reduces the chance of data-entry errors and improves interoperability. The coding system in Slicer is compatible with coding used in DICOM information objects and allows storing SNOMED-CT (Systematized Nomenclature of Human and Veterinary Medicine - Clinical Terms) or any other terminologies.
```

Comment:
I would rephrase the opening statement that the purpose of module was originally to support semantic annotation of segments in a segmentation node, which was subsequently applied to other types of annotations (if I understand correctly the following text - I have not used Terminologies with Markups myself). This is accomplished by utilizing standard coded terms that describe specific attributes of the segment: 1) general **category** of segment (e.g., ; 2) specific **type** of the segment within the broad category (e.g., ), and 3) where applicable, **anatomic region** where the segmented entity is located. Category and Type attributes can be accompanied by **modifiers** that provide more specificity in the situations where the value of the attribute corresponds to paired entities. Terminologies module follows the conventions for the attributes that accompany Segmentation objects as defined in the DICOM standard (see http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.8.20.4.html#table_C.8.20-4). 

While each of the attributes above can be populated using codes from user-defined sources, and can come from different sources of coded terms, Terminologies module provides pre-populated pick-lists, called **contexts**, that contain terms extracted from the DICOM standard, which in turn relies on the SNOMED-CT ontology.

Reply:
Can you please suggest an edit that we can simply paste? It would be much more efficient than me trying to understand your logic then do some iterations until it's fine with you. Thanks.